### ISSUE TITLE
`leading-full` not supported anymore

### ISSUE THREAD

**Issue by patrick91** [maizzle/tailwindcss-mso#31]
Just had [CI job failing with this error](https://github.com/pythonitalia/pycon/actions/runs/7365612144/job/20047140979):

```
✖ CssSyntaxError: /home/runner/work/pycon/pycon/email-templates/src/assets/css/custom/reset.css:9:3: The `leading-full` class does not exist. If `leading-full` is a custom class, make sure it is defined within a `@layer` directive.

   7 | img {
   8 |   border: 0;
>  9 |   @apply max-w-full leading-full align-middle;
     |   ^
  10 | }
  11 | 
```

This happened after the latest release was dropped (1.4.0), not sure if is our code being wrong or not, but I wanted to mention in case there was any other side effect (or other people got the same issue)

**Comment by cossssmin** [comment]
Shouldn’t be related though, `leading-full` should just be coming from the `spacing` scale in your Tailwind config - does that have `full: '100%'`?

**Comment by cossssmin** [comment]
Also what Maizzle and Tailwind versions are you running?

**Comment by cossssmin** [comment]
OK, just checked and something's definitely wrong.

`leading-full` uses `theme('width')` which does include `full: '100%'`. Removing `tailwindcss-mso` from the Tailwind config does get it working again and so does pinning it to `1.3.0`.

So sorry for this, will try to fix it as soon as I can. Meanwhile I'm seeing you already pinned it to `1.3.0` in your project, can you confirm that it works with that on your end?

**Comment by patrick91** [comment]
yup! that works fine! no rush on this btw ☺️

Patrick Arminio

**Comment by cossssmin** [comment]
Think it's an ESM/CJS conflict, changed the plugin to use ESM with import/export when I migrated it to Vitest and it's being `require`d in the CJS Tailwind config. Hate this crap, sorry for causing you trouble in the last weekend of the year 🫣

Will revert and fix it tomorrow.

**Comment by cossssmin** [comment]
Went and reverted the ESM code and published `v1.4.1`, should work fine now. Again, sorry for this!

**Comment by cossssmin** [comment]
Closing as resolved, do let me know if you encounter any issues with `v1.4.1` 👍
